sync-data-to-another-database/plugin/manifest.json

```diff
@@ -2,5 +2,5 @@
   "name": "kuzzle-plugin-sync-cassandra",
   "version": "1.2.0",
   "threadable": false,
-  "kuzzleVersion": "^1.1.x"
+  "kuzzleVersion": "^1.1.11"
```

Comment:
`1.2.11`?

Reply:
We need the new `m*` routes if we want to insert batch of 50 000 documents